setup.py

```diff
@@ -25,7 +25,7 @@
 
 setup(
     name='testing.postgresql',
-    version='1.3.0',
+    version='1.3.1',
```

Comment:
This would be a minor release, not a patch release, but the version bump should be left out of the PR anyway.